.vscode/settings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "cmake.configureEnvironment": {
+    "VCPKG_ROOT": "",
+    "VCPKG_DEFAULT_TRIPLET": ""
+  },
+  "cmake.configureSettings": {
+    "WARNINGS_AS_ERRORS" : "ON",
+    "TRANSPORT_CURL" : "OFF",
+    "UNIT_TESTING" : "OFF",
```

Comment:
> will help users debug and run samples and tests.

If the primary benefit here is to debug and run samples/tests, unit testing should be "ON", no?